run_brer/tests/test_run_config.py

```diff
@@ -69,6 +69,8 @@ def test_run_config(tmpdir, data_dir):
         # Include a test for kwarg handling.
         rc.run(threads=num_cpus)
 
```

Comment:
For the places that we have to set non-zero alpha in testing, let's also (first) test that the error is correctly thrown.

Suggestion to insert the following at this line.
```python
# Confirm that our sanity checks work.
with pytest.raises(ValueError):
    # Let the untrained value from the high-tolerance training short-cut trigger a sanity check.
    rc.run()
```

Reply:
Placing your suggestion after `kwarg handling`? 

`# Include a test for kwarg handling.
        rc.run(threads=num_cpus)
        # Confirm that our sanity checks work.
        with pytest.raises(ValueError):
            # Let the untrained value from the high-tolerance training short-cut trigger a sanity check.
            rc.run()
        # Include a test to override alpha=0 fail
        rc.run_data.set(alpha=1.0)`

Reply:
Sorry, I have GOT to figure out how to make the code look all shiny and pretty in comments.

Reply:
This still needs a `with pytest.raises` to catch the exception.